rubocop-solidus.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |spec|
   DESCRIPTION
   spec.homepage = 'https://www.github.com/nebulab/rubocop-solidus'
   spec.license = 'MIT'
-  spec.required_ruby_version = '>= 2.4.0'
```

Comment:
Are you sure about it? 
I would like to use this extension also on old solidus version. 
I know that ruby versions lower than 2.7 are not supported, but it looks to me that this extension is working also with old versions.

Reply:
Added Ruby 2.4 support in 90d4f88fa6b029ef4b2fd97bbed6503174c03cf7